### PR TITLE
Zeal branch

### PR DESCRIPTION
**Pull request by zpatel-dev**
Added create game, join the game, resume game.

### CODE REVIEW

File: mysite/templates/game/css/checkers_style.css [hunk 1]
```diff
@@ -148,46 +148,46 @@ span {
...
-
-    .dark-piece {
-        height: 1.5vw;
-        width: 1.5vw;
-    }
-
-    .light-turn-text {
-        font-size: 35px;
-    }
+/* @media screen and (max-width: 380px) { */
```
Comment (brucker3) [inline mysite/templates/game/css/checkers_style.css]: dont submit commeted code. please remove

File: mysite/manage.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 """Django's command-line utility for administrative tasks."""
 import os
 import sys
-from mysite import start_redis_server
+#from mysite import start_redis_server
```
Comment (brucker3) [inline mysite/manage.py:5]: should this be added back in ?

File: mysite/checkers/player.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from .game import *
```
Comment (brucker3) [inline mysite/checkers/player.py]: to be removed

File: mysite/manage.py (hunk 2 above)
Comment (zpatel-dev) [reply mysite/manage.py:5]: let it be commmented out so we some one want to use auto startup of redis they can use.

File: mysite/templates/game/css/checkers_style.css (hunk 1 above)
Comment (zpatel-dev) [reply mysite/templates/game/css/checkers_style.css]: done

File: mysite/checkers/player.py (hunk 3 above)
Comment (zpatel-dev) [reply mysite/checkers/player.py]: done

**Review by brucker3 — APPROVED** [review]

**Pull request merged**